core/src/main/scala/kafka/coordinator/transaction/TransactionMarkerChannelManager.scala

```diff
@@ -301,6 +301,10 @@ class TransactionMarkerChannelManager(config: KafkaConfig,
           // enqueue for retry
           txnLogAppendRetryQueue.add(txnLogAppend)
 
+        case Errors.COORDINATOR_LOAD_IN_PROGRESS =>
```

Comment:
Good catch! I noticed that in `TransactionStateManager.loadTransactions()`, we iterate through the loaded transactions and call `TransactionMarkerChannelManager.addTxnMarkersToSend` before we remove the partition from `loadingPartitions`. Is it possible for the markers to be successfully written before that time (i.e. while the partition is still being loaded)?